src/Libraries/CoreNodeModels/Selection.cs

```diff
@@ -103,7 +103,10 @@ public virtual string SelectionSuggestion
             this.selectionType = selectionType;
             this.selectionObjectType = selectionObjectType;
 
-            OutPortData.Add(new PortData("Elements", Resources.SelectionPortDataResultToolTip));       
+            
+            string portName = selectionType.ToString().Equals("One")? "Element" : "Elements";
```

Comment:
@ke-yu  I want to check its value. if I dont use .Tostring(), then I have to create a new variable and assign value to it.